python/segyio/open.py

```diff
@@ -172,8 +172,14 @@ def open(filename, mode="r", iline = 189,
     )
 
     try:
+        # Bytes 215-216
+        delay_scalar = f.header[0][segyio.TraceField.ScalarTraceHeader]
+        if delay_scalar == 0:  # A value of zero is assumed to be a scalar value of 1.
+            delay_scalar = 1
+        elif delay_scalar < 0:  # if negative, scalar is used as divisor.
```

Comment:
Maybe remove these comment too. When I first read it I thought you made this assumption, but while checking the spec I see this is the intended behaviour. And the second one just explaining what's going on in the next line of code anyways

Reply:
Cleaner now 👍